djstripe/managers.py

```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-
+"""Model managers related to dj-stripe."""
```

Comment:
Strange that this doesn't have a full module doc. Can you please add it?

Authors are me, pydanny and wahuneke
https://github.com/kavdev/dj-stripe/blame/%23162-api-updates-through_2015-07-28/djstripe/managers.py